src/controllers/validation/locations.js

```diff
@@ -33,8 +45,9 @@ export default {
       pageNumber: Joi.number(),
       pageSize: Joi.number(),
       age: Joi.number(),
+      sortBy: Joi.string(), // TODO: specify three possible values
```

Comment:
If I'm not mistaken, it would only require:
```suggestion
      sortBy: Joi.string().valid(SORT_BY_OPTIONS),
```

Reply:
I think it wants:

```
 sortBy: Joi.string().valid.apply(Joi, SORT_BY_OPTIONS),
```

According to the API documentation: https://joi.dev/api/?v=17.13.3#anyvalidvalues---aliases-equal

Reply:
Fixed in 7e7cd080629f37a1e791a4ff8bf5b6519c681e14

Validated: 

![image](https://github.com/user-attachments/assets/b6520e2d-7452-406e-b979-3b31eb7aa858)

Reply:
I imagine that works too, but you can see it's not necessary, e.g. [here](https://github.com/streetlives/streetlives-api/blob/develop/src/controllers/validation/services.js#L55).

Actually in the doc you shared it also uses chaining, without needing to bind, so maybe I'm not understanding what you mean.
![image](https://github.com/user-attachments/assets/8394f416-0621-41d7-8f4d-33e13b118460)

Reply:
You mean because it typically accepts spread args and not an array? Judging by [this code](https://github.com/streetlives/streetlives-api/blob/develop/src/controllers/validation/services.js#L55), an array should work too. If not, should fix that one as well.

(And if it _is_ necessary to spread, probably simpler nowadays to just do `sortBy: Joi.string().valid(...SORT_BY_OPTIONS)`.)

Reply:
P.S. Either way you add this, the custom validation in the controller can be removed now.

Reply:
> You mean because it typically accepts separate args and not an array? Judging by [this code](https://github.com/streetlives/streetlives-api/blob/develop/src/controllers/validation/services.js#L55) in validation for service routes, an array should work too. If not, we should fix that one as well.
> 
> (And if it _is_ necessary to spread, probably simpler nowadays to just do `sortBy: Joi.string().valid(...SORT_BY_OPTIONS)`.)

Fixed in 7d09c2a2a145b0a4845a70f2ff1d7dc07c4f3e4e

Reply:
> P.S. Either way you add this, the custom validation in the controller can be removed now.

Fixed in bf6b5ec80309d0795b14078f4bf81aacc1b5879c